projects/go-lib/src/lib/components/go-table/go-table.component.spec.ts

```diff
@@ -918,6 +928,15 @@ describe('GoTableComponent', () => {
       expect(component.renderTable).toHaveBeenCalled();
     });
 
+    it('sets the searchTerm if a searchTerm is passed in to the config', () => {
+      component.tableConfig.searchConfig.searchTerm = 'koala';
+      component.tableConfig.searchConfig.searchable = true;
+
+      component.ngOnInit();
+
+      expect(component.searchTerm.value).toEqual(component.tableConfig.searchConfig.searchTerm);
```

Comment:
should we also add a validation here to make sure it actually filters it down here as well as just verifying that the search term shows up?

Reply:
Yeah, I've added that.